clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/VeniceChangelogConsumerImpl.java

```diff
@@ -248,6 +248,52 @@ public CompletableFuture<Void> seekToEndOfPush(Set<Integer> partitions) {
     return internalSeek(partitions, topic, kafkaConsumer::seekToBeginning);
   }
 
+  @Override
+  public void pause() {
+    this.pause(
+        kafkaConsumer.assignment()
```

Comment:
Why is this tightly coupled to Kafka, rather than the pub sub abstractions?

Reply:
[I'm waiting on a PR from Hao to get this out](https://github.com/linkedin/venice/pull/395).  The Pubsub abstraction has pause, so should be a one line replacement I think.